andela/mont-blanc-cfh
# #151991932 Implement JWT for succesful login

#### What does this PR do?
Assign JWT to user on successful login
#### Description of Task to be completed?
On successful login, a token is generated for the user side and stored in local storage for authorizing future requests
#### How should this be manually tested?
User should login with his/her details and check the browser's local storage if token exists
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#151991932
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/9434249/31676445-89742e1e-b35f-11e7-9e35-45171c2dc8f5.png)

#### Questions:
None
